README.md

```diff
@@ -218,7 +218,7 @@ You will need all the following in your deployment environment (eg, your laptop)
 | [git](https://git-scm.com/)                     | 2.17+                  | `git --version`           |
 | [Maven](https://maven.apache.org/)              | 3.6+                   | `mvn -v`                 |
 | [Java JDK 11+](https://openjdk.org/install/) | 11, 17, 21 (see notes) | `mvn -v &#124; grep Java` |
-| [Terraform](https://www.terraform.io/)          | 1.3.x, <= 1.6          | `terraform version`       |
+| [Terraform](https://www.terraform.io/)          | 1.3.x, <= 1.7          | `terraform version`       |
```

Comment:
From YAML https://github.com/Worklytics/psoxy/pull/693/files#diff-224777d94940d8246633bd46b1a26f8cf36bc33a4ee0046fb9891c66d55f1fc0L24 version is `~1.7.0`. 

Maybe .. ?

```suggestion
| [Terraform](https://www.terraform.io/)          | 1.3.x, <= 1.7.x          | `terraform version`       |
```